app/src/main/java/com/lagradost/cloudstream3/plugins/PluginManager.kt

```diff
@@ -527,7 +608,8 @@ object PluginManager {
 
     private fun createNotification(
         context: Context,
-        extensionNames: List<String>
+        extensionNames: List<String>,
+        @StringRes title: Int
```

Comment:
if you are going to do this but also support formatting use UiText instead, https://github.com/recloudstream/cloudstream/blob/28b4456dfd6e808553a7bbffbd06c8ef4e6dfc9a/app/src/main/java/com/lagradost/cloudstream3/ui/result/UiText.kt#L43

Reply:
The string resource needs a decimal input (list size), and the UI text flattens the list entries so it doesn't match

Reply:
Yeah Ik, I meant inside UIText.toString(), the method flattens the list and not doing ``extensionNames.size``. It will always error cause wrong parameter is used. Will have to check more about UiText().